Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -32,13 +32,13 @@ const Contest = () => {
     const { state: { user: { permissions: { canAccessAdministration } } } } = useAuth();
     const { actions: { setPageTitle } } = usePageTitles();
 
-    const navigationContestClass = 'navigationContest';
+    const navigationContestClass = styles.navigationContest.toString();
```

Comment:
1. `toString` should not be necessary
2. I think this class is used for tests, so it must remain a `string`. Otherwise, it will be with autogenerated name skip the caching of selectors. @MadjarovaML please confirm

Reply:
It was unused

Reply:
Yes, it is used in tests